src/sql/workbench/browser/modelComponents/media/toolbarLayout.css

```diff
@@ -50,8 +50,8 @@
 }
 
 .modelview-toolbar-container .modelview-toolbar-component modelview-button .monaco-text-button.icon {
-	padding-left: 15px;
-	background-size: 11px;
+	padding-left: 19px;
+	background-size: 15px;
```

Comment:
which features are using now? need to make sure the feature owners are aware of this.

Reply:
The only other place I see this being used in our repo is schema compare and that looks fine 

![image](https://user-images.githubusercontent.com/28519865/66871689-082ff800-ef59-11e9-8566-4e49dd780c1c.png)

For comparison here's how it looks right now :
![image](https://user-images.githubusercontent.com/28519865/66871762-3b728700-ef59-11e9-8541-197c47b9ca6f.png)

@kisantia @udeeshagautam - FYI in case you see anything looking off or don't think the larger icons are appropriate.